plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginCommandHandler.java

```diff
@@ -6,15 +6,18 @@
 import org.eclipse.ui.IWorkbenchWindow;
 import org.eclipse.ui.handlers.HandlerUtil;
 import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.window.SameShellProvider;
 
 public class GoogleLoginCommandHandler extends AbstractHandler {
 
-	public Object execute(ExecutionEvent event) throws ExecutionException {
-	  boolean success = new GoogleLoginTemporaryTester().testLogin();
-
+  public Object execute(ExecutionEvent event) throws ExecutionException {
     IWorkbenchWindow window = HandlerUtil.getActiveWorkbenchWindowChecked(event);
```

Comment:
That's handy. However, this line is auto-generated by a wizard for a menu command, and it gets a window based on the `event` argument. so I feel like I don't need to touch it?